lib/nosedrum/application_command.ex

```diff
@@ -269,6 +269,18 @@ defmodule Nosedrum.ApplicationCommand do
   """
   @callback options() :: [option]
 
+  @doc """
+    An optional callback that returns a bitset for the required default permissions to run this command.
+
+    Example callback that requires that the user has the permission to ban members to be able to see and execute this command
+
+    ```elixir
+    def default_member_permissions, do:
+      Nostrum.Permission.to_bitset([:ban_members])
+    ```
+  """
```

Comment:
Let's dedent this by two spaces so it matches the rest of the module 

```suggestion
  @doc """
  An optional callback that returns a bitset for the required default permissions to run this command.

  Example callback that requires that the user has the permission to ban members to be able to see and execute this command

  ```elixir
  def default_member_permissions, do:
    Nostrum.Permission.to_bitset([:ban_members])
  ```
  """
```

Reply:
I'll need a bit more time to do your recommended changes for personal reasons, but it's noted!